SAP/spartacus
# feat: Replace `@import` with `@use` when importing styles in SPA app

This PR addresses helps to avoid deprecation warnings generated by the Dart Sass version used by the Angular 19 app that appears when building the app with Spartacus.
![image](https://github.com/user-attachments/assets/6b1dd091-dbdd-49f5-b94e-6499eb6b31dd)

NOTE: `@import` is still used in the internal Spartacus`.scss` files and such a refactor is out of the scope of this PR as well as the "upgrade to Angular 19" topic

QA (installation script):  
- change version of all packages to `2212.32.0` (temporary test value)
- build and deploy Spartacus packages locally:
  `npx ts-node ./tools/schematics/testing.ts` in the SPA root folder`
- create new app `npx @angular/cli@19 new my-app --standalone=false --style=scss --routing=false`
- go to the project's directory and install Spartacus from local packages:
  `npx @angular/cli@17 add @spartacus/schematics@latest --baseUrl="https://40.76.109.9:9002"`
- run `npm start` and verify that there is no error similar to the one from the description 
- run `npm build` and verify that there is no error similar to the one from the description
